src/js/tech/tech.js

```diff
@@ -238,6 +238,14 @@ class Tech extends Component {
    * @method dispose
    */
   dispose() {
+    // clear out text tracks because we can't reuse them between techs
```

Comment:
It would mean that users would need to re-add `cuechange` events to the text tracks each time the tech changes. Also, events that are added to the cues themselves would need to be re-added, though, we don't currently support that for the emulated cues.